test/protocol-buffer-conversion-test.ts

```diff
@@ -1,6 +1,27 @@
 import { assert } from 'chai'
 import XRPCurrency from '../src/xrp-currency'
+import XRPPathElement from '../src/xrp-path-element'
+import XRPPath from '../src/xrp-path'
 import { Currency } from '../src/generated/web/org/xrpl/rpc/v1/amount_pb'
+import { Payment } from '../src/generated/web/org/xrpl/rpc/v1/transaction_pb'
+import { AccountAddress } from '../src/generated/web/org/xrpl/rpc/v1/account_pb'
+
+// TODO(amiecorso): Refactor these to separate files.
+const testCurrencyProto: Currency = new Currency()
+testCurrencyProto.setCode(new Uint8Array([1, 2, 3]))
+testCurrencyProto.setName('currencyName')
+
+const testAccountAddress = new AccountAddress()
+testAccountAddress.setAddress('r123')
+
+const testAccountAddressIssuer = new AccountAddress()
+testAccountAddressIssuer.setAddress('r456')
+
+// Populate test PathElement
+const testPathElement = new Payment.PathElement()
+testPathElement.setCurrency(testCurrencyProto)
+testPathElement.setAccount(testAccountAddress)
+testPathElement.setIssuer(testAccountAddressIssuer)
```

Comment:
Is this effectively a `beforeAll` that you need to run before your tests?

Does our test library here have a `beforeAll` method? If not, could you add some comments here to clarify that this is setup (if it is) and why this is necessary?

Reply:
So apparently Mocha does have a before method... and a beforeEach, but these don't seem to put variables in the proper scope.  I've spent a ton of time trying to get them to work and I think it's clearer and simpler to just have some code above the describe method with an additional comment.

Reply:
That's reasonable. I just ran into the variable scoping with Mocha as well. It makes sense, given what JS scoping is, but definitely annoying.

I still managed to use a `before/after` block, because I basically do:
`const app = new Express()` on the outside, so it's in scope, and then I have:
```
before() {
  app.init();
}

after() {
  app.close();
}
```

But keeping everything outside is just as reasonable.